lib/ip.go

```diff
@@ -37,34 +50,93 @@ func RandIP4List(n int) []net.IP {
 	return ips
 }
 
-// RandIP4Range returns a list of randomly generated IPv4 addresses within
-// starting and ending IPrange.
+// EvalIP4Range checks if the starting and ending IP range is valid or not.
+//
+// note: starting and ending IPs must be valid IPv4 string formats.
+func EvalIP4Range(
```

Comment:
This is not actually a validation, it's a generation. This should be called `NewIP4Range`, and it should match the documentation form of `NewIP`.